servers/jax-rs-tests/src/main/java/org/projectnessie/jaxrs/AbstractTestRest.java

```diff
@@ -2089,6 +2089,29 @@ public void testDiff() throws BaseNessieClientServerException {
             });
   }
 
+  @Test
+  public void commitLogExtendedForUnchangedOperation() throws Exception {
```

Comment:
maybe just include this into the `commitLogExtended` test?

Reply:
I cannot add extra validation in existing `commitLogExtended` testcase as operations will have put and delete.
I added one unchanged operation now to that testcase.

I specifically wanted to check with single delete operation to be sure that we don't refer that operation somewhere else which can cause NPE. Hence a separate test case